prisma/schema.prisma

```diff
@@ -7,27 +7,32 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Post {
-  id        Int      @id @default(autoincrement())
-  title     String   @db.VarChar(255)
-  createdAt DateTime @default(now()) @db.Timestamp(6)
-  content   String?
-  published Boolean  @default(false)
-  authorId  Int
-  User      User     @relation(fields: [authorId], references: [id])
+model User {
+  id         Int         @id @default(autoincrement())
+  username   String
+  hash       String
+  chapters   Chapter[]
+  admins     Admin[]
+  recipients Recipient[]
 }
 
-model Profile {
-  id     Int     @id @default(autoincrement())
-  bio    String?
-  userId Int     @unique
-  User   User    @relation(fields: [userId], references: [id])
+model Chapter {
+  id          Int     @unique @default(autoincrement())
```

Comment:
use the `@id` annotation here instead of `@unique`. We are using PostgreSQL as the underlying database so we need to set primary keys.